content/community/contribute/how_to_help.md

```diff
@@ -6,6 +6,8 @@ menu:
         weight: 1
 ---
 
+<!-- TODO: refactor this page to be about libp2p -->
```

Comment:
Not critical, but completing this TODO as part of this PR would present a consistent libp2p-focused site.  It's the only page left where there are still significant holdover from the IPFS docs repo.  Your call, though.